blocks/edit/prose/index.js

```diff
@@ -179,16 +179,20 @@ function restoreCursorPosition(view) {
 }
 
 function addSyncedListener(wsProvider) {
-  let initialContentLoaded = false;
-  wsProvider.on('synced', (isSynced) => {
-    if (isSynced && !initialContentLoaded) {
-      const pm = document.querySelector('da-content')?.shadowRoot
-        .querySelector('da-editor')?.shadowRoot.querySelector('.ProseMirror');
-      if (pm) pm.contentEditable = 'true';
-
-      initialContentLoaded = true;
+  const handleSynced = (isSynced) => {
+    if (isSynced) {
+      const permissions = document.querySelector('da-title').permissions || [];
```

Comment:
The permissions are already passed into `initProse` you can pass them into `addSycnedListener`.